libraries/rush-lib/src/utilities/performance.ts

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+// See LICENSE in the project root for license information.
+
+import type { PerformanceEntry, PerformanceMark, PerformanceMeasure } from 'node:perf_hooks';
+
+/**
+ * Starts a performance measurement.
```

Comment:
There are probably a bunch of cases where doing a pair of these with a disposable would be useful.